requirements.txt

```diff
@@ -4,8 +4,10 @@ Flask-Babel
 itsdangerous==1.1.0
 Jinja2==2.10.3
 MarkupSafe==1.1.1
-Werkzeug==0.16.0
+werkzeug
 nose
 wiktextract==0.2.0
 elasticsearch>6.0.0,<7.0.0
 elasticsearch-dsl>6.0.0,<7.0.0
+lxml
```

Comment:
I don't think this new dependency is necessary; the module that imports lxml is copy-and-pasted from core and lxml isn't actually used.